src/System.Private.ServiceModel/src/Resources/System.Private.ServiceModel.rd.xml

```diff
@@ -59,6 +59,11 @@
           </Method>
         </Type>
       </Namespace>
+      <Namespace Name="System.ServiceModel.Dispatcher">
+          <Type Name="FaultFormatter">
+            <Type Name="OperationFault{T}" Dynamic="Required All" />
```

Comment:
Ah, so this is the syntax for nested classes.  Good to know.  This comment originated from an experimental version of CodeFlow